bin/update_envoy.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Update the Envoy SHA in istio/proxy WORKSPACE with the first argument
+set -e
+
+UPDATE_BRANCH=${UPDATE_BRANCH:-"master"}
+
+SCRIPTPATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOTDIR=$(dirname "${SCRIPTPATH}")
+cd "${ROOTDIR}"
+
+# Get ENVOY_SHA256
+SHA256=$(wget https://github.com/istio/envoy/archive/$1.tar.gz && sha256sum $1.tar.gz)
```

Comment:
repo and org here should be derived from workspace file: https://github.com/istio/proxy/blob/12cfbda324320f99e0e39d7c393109fcd824591f/scripts/generate-wasm.sh#L49

Reply:
updated!